ultraScurve.py

```diff
@@ -156,6 +162,7 @@
         startScanModule(ohboard, options.gtx, useUltra = True, debug = options.debug)
         scanData = getUltraScanResults(ohboard, options.gtx, SCURVE_MAX - SCURVE_MIN + 1, options.debug)
         for i in range(0,24):
+            if (mask >> i) & 0x1: continue
```

Comment:
Commenting out line 165 generates exception:

[dorney@cosmicstandtif]~/scratch0/CMS_GEM/CMS_GEM_DAQ/testing/validation% ultraScurve.py -s2 -g0 --shelf=3 --vfatmask=0xF40400 2>&1 | tee scurveLog.txt
2017.06.21.17.45
Link 0 T1 controller status: 2
21-06-17 17:46:01.952104 [7fae214d8700] ERROR - Returned Header from URI "ipbustcp-2.0://192.168.1.42:60002", 0x2DF70104 ( transaction id = 0x00000DF7, transaction type = 0x00, word count = 1 ) had response field = 0x04 indicating an error (1 bytes into IPbus reply payload)
21-06-17 17:46:01.952148 [7fae214d8700] ERROR - Original sent header was 0x2DF7010F, for base address 0x00500E7A, 1 bytes into IPbus send payload
No handlers could be found for logger "gempython.tools.vfat_user_functions_uhal"
Channel #0
GEM_AMC.OH.OH0.ScanController.ULTRA
FW scan mode       : 3
FW scan min        : 0
FW scan max        : 254
Ultra FW scan mask : 0x00f40400
FW scan channel    : 0
FW scan step size  : 1
FW scan n_triggers : 1000
FW scan status     : 0x00000000
An exception occurred gem.shelf03.amc02.optohybrid00: error on VFAT transaction (chip 10, ContReg3)

Whereas running with line 165 uncommented produces expected behavior.  Note test stand used has dead I2C sectors for VFAT Pos 20-23 and no connector on GEB for Pos 10 and 18.